boards/arm/stm32l496g_disco/doc/stm32l496g_disco.rst

```diff
@@ -172,8 +172,8 @@ Default Zephyr Peripheral Mapping:
 System Clock
 ------------
 
-STM32L496G Discovery System Clock could be driven by internal or external oscillator,
-as well as main PLL clock. By default System clock is driven by PLL clock at 80MHz,
+STM32L496G Discovery System Clock could be driven by an internal or external oscillator,
```

Comment:
Fixing the grammar in this paragraph is a good change, but the Zephyr policy is to split commits up into single, logical changes, as described in the contribution guide:

http://docs.zephyrproject.org/contribute/contribute_guidelines.html#commit-guidelines

Since this commit is about openocd support, could you split this change up into a different commit?